device/dell/x86_64-dell_z9100_c2538-r0/plugins/eeprom.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+#############################################################################
+# Dell Z9100
+#
+# Platform and model specific eeprom subclass, inherits from the base class,
+# and provides the followings:
+# - the eeprom format definition
+# - specific encoder/decoder if there is special need
+#############################################################################
+
+try:
+    import eeprom_tlvinfo
```

Comment:
My mistake. I didn't see #342 before I raised this... I can decline this in favor of that one.